pdf/pdfcanvas.cpp

```diff
@@ -113,9 +113,16 @@ PDFCanvas::PDFCanvas(QQuickItem* parent)
 
 PDFCanvas::~PDFCanvas()
 {
-    if(d->resizeTimer)
-        delete d->resizeTimer;
+    for( int i = 0; i < d->pageCount; ++i )
```

Comment:
Should be like...
``` 
for (int ... ++i) {
}
``` 
The rests seems to follow more or less the style you used. So I'm fine with that as well. One could do a commit that fixes all coding style related issues in a commit. Let's do not mix it up over here.